src/data/articles.yaml

```diff
@@ -0,0 +1,19 @@
+- title: Wiki info about the city of Portland, OR
+  author: Author Name
+  date: May 5th, 2016
```

Comment:
Agreed. The `{{timestamp}}` helper is probably the better place to handle changing up the format.